### DISCUSSION TITLE
help!!!!

### DISCUSSION THREAD

**Discussion by lee19928** [nextcloud/all-in-one#5100] [Questions] [answered]
![00](https://github.com/user-attachments/assets/e0f68335-3f6a-4c1f-85d4-add0833f6bf6)
![99](https://github.com/user-attachments/assets/dd3f47e7-b0ac-4b4c-a354-a72621e079ff)
![88](https://github.com/user-attachments/assets/54a88630-fa4d-430d-8d14-bc75d835c187)
what is problem，help。

**Comment by lee19928** [comment]
p1：Verify that the installation is complete
P2：is my caadyfile config 
p3:is caady contianer error info

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Reply by lee19928** [reply]
i already see this link and check ，Still can't find the problem，the http code 400

**Comment by lee19928** [comment]
OK， i‘ll&nbsp; check the link ，and can you help me check the docker compose file？ many thanks


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "nextcloud/all-in-one"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月7日(星期三) 晚上7:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [nextcloud/all-in-one] help!!!! (Discussion #5100)





 
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Comment by lee19928** [comment]
hi，Is this an error?


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "nextcloud/all-in-one"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月7日(星期三) 晚上7:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [nextcloud/all-in-one] help!!!! (Discussion #5100)





 
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Comment by lee19928** [comment]
hi&nbsp; this is my&nbsp;topological ，can you help me to fix ？I really need nextcloud server。If you have time, you can control my computer remotely 。thanks


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "nextcloud/all-in-one"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月7日(星期三) 晚上7:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [nextcloud/all-in-one] help!!!! (Discussion #5100)





 
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Comment by szaimen** [comment, marked as answer]
It needs to be `reverse_proxy http://localhost:11000` instead of `reverse_proxy https://localhost:11000`.

**Comment by lee19928** [comment]
My caddyfile set this ，is correct？


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "nextcloud/all-in-one"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月9日(星期五) 下午3:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [nextcloud/all-in-one] help!!!! (Discussion #5100)





 
It needs to be reverse_proxy http://localhost:11000 instead of reverse_proxy https://localhost:11000.
